src/components/process/manage/ManageProcess.js

```diff
@@ -42,14 +42,6 @@ export const ManageProcess = () => {
         })
     }
 
-    const handleSubmit = () => {
```

Comment:
why is that so?

Reply:
Nie wiem jak się to znalazło w tym diffie